### ISSUE TITLE
train_flux.py在npu上训练，vit中的torch.nn.Conv2d算子运行时间特别长，导致flux训练无法正常进行

### ISSUE THREAD

**Issue by yyyyrrrrif** [ByteDance-Seed/VeOmni#510]
### System Info

cann版本：8.5.0
veomni: master分支 afe8324a

### Information

- [ ] The official example scripts
- [ ] My own modified scripts

### Tasks

- [ ] An officially supported task in the `examples` folder (such as GLUE/SQuAD, ...)
- [ ] My own task or dataset (give details below)

### Reproduction

bash train.sh task/omni/train_flux.py configs/dit/flux_sft.yaml \
  --train.init_device npu \
  --train.micro_batch_size 1 \
  --train.global_batch_size 2 \
  --train.enable_fsdp_offload true \
  --train.enbale_activate_offload true

### Expected behavior

flux模型可正常在npu上进行训练

**Comment by Luosuu** [comment]
@Crystal-jiang
